### PR TITLE
Update to the maistra docs for maistra 0.7.0: MAISTRA-195

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

**Comment by knrc** [comment]
@brian-avery @geekspertise Can you please take a look at this?

**Comment by knrc** [comment]
Let me fix the problem first

File: topics/docs/custom-install.adoc [hunk 1]
```diff
@@ -58,6 +58,22 @@ spec:
       filter: booster.mission.metadata.istio
       branch: v71
       repo: https://github.com/fabric8-launcher/launcher-booster-catalog.git
+  threeScale:
```
Comment (brian-avery) [inline topics/docs/custom-install.adoc]: Would it be better to call this 3scale to be consistent with the product name? Seems to check out on yaml validators and I don't see anything in the spec saying it's not allowed.

Comment (knrc) [reply topics/docs/custom-install.adoc]: In lots of other places we are using threescale so it is likely we will change it in the next sprint to be consistent with that.

**Pull request merged**